docs/_documentations/troubleshooting.md

```diff
@@ -773,3 +774,31 @@ Info added in 0.10.0.
 To assist with problem determination, raise the default Codewind server log level to **Debug** or **Trace**. Use the `cwctl loglevels` command or follow the instructions for an IDE:
 - In Eclipse, enable support features in the Codewind preferences, then right-click the connection in the Codewind Explorer view and click **Codewind server log level**.
 - In VS Code, use the **Codewind: Set Codewind Server Logging Level** command in the Command Palette.
+
+# Executable file not found on PATH in VS Code
+<!--
+Action/Topic: Executable file not found on PATH in VS Code
+Issue type: info
+Issue link: https://github.com/eclipse/codewind/issues/1682
+Solution: https://github.com/eclipse/codewind/issues/1682#issuecomment-572649328
+Info added in 0.11.0.
+-->
+Codewind will not work if it cannot find the `docker` executable in the [`PATH`](https://en.wikipedia.org/wiki/PATH_(variable)). You will see an error message like the following:
+```
+exec: "docker": executable file not found in $PATH
+```
+First, make sure `docker` is installed and can be run from the command line. The terminal used does not have to be a terminal within VS Code. Try running:
+```
+$ docker --version
+Docker version 19.03.8, build afacb8b
+```
+and make sure the output is similar to above.
+
+If you can run `docker` from the command line, but Codewind still fails to find `docker`, it's possible that VS Code is using a different `PATH` than the terminal you used. To make sure they are using the same `PATH`, perform the following steps:
+
+1. Make sure you can run VS Code (the executable is `code`) from the command line.
```

Comment:
What if we put "the executable is `code`" in its own sentence? "Make sure you can run VS Code from the command line. The VS Code executable is `code`."